### PR TITLE
Enable spell checking fix

### PR DESCRIPTION
**Pull request by tohmjudson**
Added spell checking to lint process.

### CODE REVIEW

**Review by jasonLaster — COMMENTED** [review]
This looks good if we can run it in debugger.html.

**Comment by wooorm** [comment]
Looks good, but have you linked this into `debugger.html` to check if that introduces lots of warnings?

**Comment by tohmjudson** [comment]
I think with the linking issue fix (debugger.html/package.json scripts update to `"lint-md": "remark -u devtools-linters/markdown/preset -qf *.md src configs docs",` and adding devtools-linters to the dependencies) and this, we are in good shape. The dictionary.txt has all of the currently caught misspellings when running `yarn lint-md`. There are still actual misspellings in the list, but it is a good starting point

File: natural-language/dictionary.txt [hunk 1]
```diff
@@ -4,4 +4,461 @@
...
+ast
+astBreakpoint
+astbreakpoints
+astLocation
+async
+Autocomplete
+autocomplete 
+autoconf213
+autofix
+avaliable
```
Comment (jasonLaster) [inline natural-language/dictionary.txt]: oops

File: natural-language/dictionary.txt [hunk 2]
```diff
@@ -4,4 +4,461 @@
...
+Bugfix
+bugfixes
+bz
+callee
+CallSites
+callstack
+CamelCased
+camo 
+changelog
+childen
```
Comment (jasonLaster) [inline natural-language/dictionary.txt]: yikes

File: natural-language/dictionary.txt [hunk 3]
```diff
@@ -4,4 +4,461 @@
...
+ContextMenu
+ContextMenus
+cov-screen
+covariants
+css
+CSS
+ctrl
+debuggable
+debuggee
+debuggy
```
Comment (jasonLaster) [inline natural-language/dictionary.txt]: haha should be debuggee

File: natural-language/dictionary.txt [hunk 4]
```diff
@@ -4,4 +4,461 @@
...
+devtools
+DevTools
+devtools-core 
+devtools-launchpad
+devtools-reps
+devtools-source-map
+Diéssica
+dir
+disableBreakpoint
+discoverable.
```
Comment (jasonLaster) [inline natural-language/dictionary.txt]: @wooorm i thought we checked for `.`

File: natural-language/dictionary.txt [hunk 5]
```diff
@@ -4,4 +4,461 @@
...
+dispell
```
Comment (jasonLaster) [inline natural-language/dictionary.txt]: this is probably a mispelling :P

File: natural-language/dictionary.txt [hunk 6]
```diff
@@ -4,4 +4,461 @@
...
+dom
+Draggable
+Dropdown
+dropdown
+dt 
+EditorSearch
+embarressing
+embarressing
+embarressing
+enableBreakpoint
```
Comment (jasonLaster) [inline natural-language/dictionary.txt]: I never know how to spell embarrassing it's terrible

File: natural-language/dictionary.txt [hunk 7]
```diff
@@ -4,4 +4,461 @@
...
+enableBreakpoint
+es5
+ES6
+es6 
+eslint
+EventListeners
+expections
+expessions
```
Comment (jasonLaster) [inline natural-language/dictionary.txt]: :/

File: natural-language/dictionary.txt [hunk 8]
```diff
@@ -4,4 +4,461 @@
...
+ExtJS
+F1
+F2
+F3
+falsey
+fetchEventListeners
+fetchingListeners
+fframes
```
Comment (jasonLaster) [inline natural-language/dictionary.txt]: :/

File: natural-language/dictionary.txt [hunk 9]
```diff
@@ -4,4 +4,461 @@
...
+GIF
+GIFs
+gists
+github
+globals
+goodnessSquad
+greenkeeper-lockfile-update
+gThreadClient
+Guillaume
+guillemet
```
Comment (jasonLaster) [inline natural-language/dictionary.txt:175]: this is french, but i guess fine

File: natural-language/dictionary.txt [hunk 10]
```diff
@@ -4,4 +4,461 @@
...
+HTML
+Html
+HTTP
+http
+HTTPS
+https
+iframe
+infering
+integrations
+intermittents
```
Comment (jasonLaster) [inline natural-language/dictionary.txt]: :/

File: natural-language/dictionary.txt [hunk 11]
```diff
@@ -4,4 +4,461 @@
...
+GutterMenu
+hackable
+Hardcoded
+hebrew
+HitMarker
+hostname
+hostnames
+html
+HTML
+Html
```
Comment (jasonLaster) [inline natural-language/dictionary.txt]: should be HTML

**Review by jasonLaster — COMMENTED** [review]
looks good. I found a couple words that *are* misspellings though, which we should remove and fix in debugger.html

File: natural-language/dictionary.txt [hunk 12]
```diff
@@ -4,4 +4,461 @@
...
+js 
+JSON
+json
+JSX
+jsx 
+keyshortcut
+keyshortcuts
+L10N
+l10n
+labelled
```
Comment (jasonLaster) [inline natural-language/dictionary.txt]: :/

File: natural-language/dictionary.txt [hunk 13]
```diff
@@ -4,4 +4,461 @@
...
+layouunlink
+Lerna
+lerna
+Libarary
+libary
```
Comment (jasonLaster) [inline natural-language/dictionary.txt]: :/

File: natural-language/dictionary.txt [hunk 14]
```diff
@@ -4,4 +4,461 @@
...
+libs
+lifecycle
```
Comment (jasonLaster) [inline natural-language/dictionary.txt]: :/

File: natural-language/dictionary.txt [hunk 15]
```diff
@@ -4,4 +4,461 @@
...
+mozilla-centra
+mozilla-central
+MPL
+namespace
+NaN
+nav
+netmonitor
+newSource
+newSource
+newSources
```
Comment (jasonLaster) [inline natural-language/dictionary.txt:252]: @wooorm can we assume camelcased words are okay?

File: natural-language/dictionary.txt [hunk 16]
```diff
@@ -4,4 +4,461 @@
...
+newTabs
+nom
+noParse
 npm
+NUM_FRAMES_SHOWN
+nvm
+ObjectInspector
+occassionaly
```
Comment (jasonLaster) [inline natural-language/dictionary.txt]: :/

File: natural-language/dictionary.txt [hunk 17]
```diff
@@ -4,4 +4,461 @@
...
+polyfill
+postcss-bidirection
+practicle
+prefs
+prepush
+presentational
+prettyifyed
+PreviewFunction
+PrimaryPanes
+progres
```
Comment (jasonLaster) [inline natural-language/dictionary.txt]: :/

File: natural-language/dictionary.txt [hunk 18]
```diff
@@ -4,4 +4,461 @@
...
+Q2
+Q3
+Q4
+QAing
+qulity
+racey
+react-dom
+readibility
+readme
+realy
```
Comment (jasonLaster) [inline natural-language/dictionary.txt]: :/

File: natural-language/dictionary.txt [hunk 19]
```diff
@@ -4,4 +4,461 @@
...
+selectedSource
+selectedTab
+selectFrame
+selectSource
+selectSourceURL
+selectTab
+setActiveSearch
+setBreakpointCondition
+setState
+sexualized
```
Comment (jasonLaster) [inline natural-language/dictionary.txt]: uhhh... this should not be there :)

File: natural-language/dictionary.txt [hunk 20]
```diff
@@ -4,4 +4,461 @@
...
+SourceFooter
+sourcemapped
+sourcemaps
+sourcesText
+SourcesTree
+SourceTabs
+SourceText
+sourcetext
+sourcetree
+soure
```
Comment (jasonLaster) [inline natural-language/dictionary.txt]: :/

File: natural-language/dictionary.txt [hunk 21]
```diff
@@ -4,4 +4,461 @@
...
+UCOSP
+UI
+ui 
+unblackbox
+undef
+undefineds
+unecessary
+unicode
+unix-flavor
+unsused
```
Comment (jasonLaster) [inline natural-language/dictionary.txt]: :/

**Comment by jasonLaster** [comment]
Great, i'll push a new version that we can use in the debugger soon :)

**Comment by tohmjudson** [comment]
Awesome :)

**Comment by jasonLaster** [comment]
@tohmjudson it's bumped, you can now use version `0.0.2`

**Comment by tohmjudson** [comment]
Great! I'll get started on the other parts of the issue tonight

**Pull request merged**